src/main/kotlin/com/colivery/serviceaping/persistence/repository/HelpRequestRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.colivery.serviceaping.persistence.repository
+
+import com.colivery.serviceaping.persistence.entity.HelpRequestEntity
+import org.springframework.data.repository.CrudRepository
+import java.util.*
+
+interface HelpRequestRepository : CrudRepository<HelpRequestEntity, UUID> {
+
+    fun findAllByAdminUser_Id(adminUserId: String): Set<HelpRequestEntity>
+    fun findById(helpRequestId: String): HelpRequestEntity?
```

Comment:
Spring already provides this method in a more typesafe fashion in the base interface `CrudRepository`: https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/CrudRepository.html#findById-ID-